explorer/src/components/instruction/token-swap/types.ts

```diff
@@ -14,7 +14,7 @@ export const PROGRAM_IDS: string[] = [
 
 const INSTRUCTION_LOOKUP: { [key: number]: string } = {
   0: "Initialize Swap",
-  1: "Swap",
+  1: "Exchange",
```

Comment:
was this intentional?

Reply:
Yes, Bartosz wanted me to make that change, and I guess it ended up in this branch :).